src/App.svelte

```diff
@@ -9,12 +9,13 @@
 	/** @type {HTMLDivElement} */
 	let topRightContainer;
 	let courses = [];
+	let calanderInclude = {classEntries : true, tierceEntries : true};
```

Comment:
just a typo, looks good otherwise
```suggestion
	let calendarInclude = {classEntries : true, tierceEntries : true};
```